source/math/FloatTransform2D.ooc

```diff
@@ -33,29 +33,29 @@ FloatTransform2D: cover {
 	a, b, c, d, e, f, g, h, i: Float
 	operator [] (x, y: Int) -> Float {
 		result := 0.0f
+		version (safe) {
+			if (x < 0 || x > 2 || y < 0 || y > 2)
+				raise("Out of bounds in FloatTransform2D get operator")
```

Comment:
The version has the advantage that it tells you the index you incorrectly tried to access. This should say `"Out of bounds in FloatTransform2D get operator (#{x}, #{y})"` or something